README.md

```diff
@@ -13,3 +13,6 @@ cluster. See the OLM installation instructions [here](https://olm.operatorframew
 
 ![asciicast](assets/asciinema-demo.gif)
 
+## Install
+
+Official Kubectl plugin documentation includes [plugin install documentation](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#installing-kubectl-plugins). In short, find the required release on the [releases page](https://github.com/operator-framework/kubectl-operator/releases), decompress and place the kubectl-operator binary in your path. Kubectl will figure it out from there. 
```

Comment:
@flickerfly The krew index PR was approved!

The operator can now be installed via krew: `kubectl krew install operator`

Can you update this to reflect the simpler instructions?

`````suggestion
The `kubectl operator` plugin is distributed via [`krew`](https://krew.sigs.k8s.io/). To install it, run the following:
```console
kubectl krew install operator
```
`````